addons/test_mail/tests/test_mail_message.py

```diff
@@ -27,6 +27,62 @@ def setUpClass(cls):
         })
         cls.Message = cls.env['mail.message'].with_user(cls.user_employee)
 
+    def test_mail_message_empty(self):
```

Comment:
Would run as standard employee to also have ACLs in the way :) (``@users('employee')``)